src/AccessibilityInsights.SharedUx/Controls/TestTabs/ColorContrast.xaml.cs

```diff
@@ -119,13 +120,29 @@ private void RunAutoCCA(Bitmap bitmap)
 
             if (pair == null)
             {
+                Logger.PublishTelemetryEvent(
+                    TelemetryAction.ColorContrast_AutoDetect,
+                    new Dictionary<TelemetryProperty, string>
+                    {
+                        [TelemetryProperty.BitmapSize] = (bitmap.Width * bitmap.Height).ToString(CultureInfo.InvariantCulture),
```

Comment:
We usually use the initialization list syntax
```suggestion
                        {TelemetryProperty.BitmapSize, (bitmap.Width * bitmap.Height).ToString(CultureInfo.InvariantCulture)},
```